### ISSUE TITLE
Capture question chart instead of question card

### ISSUE THREAD

**Issue by NunoSempere** [QURIresearch/metaforecast#89]
In the question page, have the capture button capture the chart, not the question card

**Comment by NunoSempere** [comment]
Also, the embed link directs to a preview (https://metaforecast-9amh4tm2f-quantified-uncertainty.vercel.app/questions/embed/metaculus-9988). Is this intentional?

**Comment by berekuk** [comment]
> Also, the embed link directs to a preview (https://metaforecast-9amh4tm2f-quantified-uncertainty.vercel.app/questions/embed/metaculus-9988). Is this intentional?

Yes, it was an easiest way to add a preview, just a link to the same embeddable page.

But I have an idea how to improve it, will do a PR soon.
